apps/system/js/app_chrome.js

```diff
@@ -39,6 +39,7 @@
     this._themeChanged = false;
     this._titleTimeout = null;
     this.scrollable = app.browserContainer;
+    this._previousOrigin = '';
```

Comment:
nit: In a resting state, this is actually current origin, I'd probably call it this._currentOrigin. We would probably change this to this._currentScope once we support app scope.